src/backend/app/backend/blueprints/user.py

```diff
@@ -1,3 +1,30 @@
 from flask import Blueprint, jsonify
 
-user = Blueprint('user', __name__)
\ No newline at end of file
+user = Blueprint('user', __name__)
+
+@user.route('/get_data')
+def get_data():
```

Comment:
Are you calling this function? I did not receive any backend calls.
![image](https://user-images.githubusercontent.com/44426592/79142186-c545c480-7dc3-11ea-80fb-cab9feed0f92.png)